README.md

```diff
@@ -10,4 +10,5 @@ Repository for the course "HTML coding for kids"
 2. [Структура HTML-документа](https://github.com/rakavets/html-coding/blob/chapter2/chapter2/technical-task.md)
 3. [Разметка текста](https://github.com/rakavets/html-coding/blob/chapter3/chapter3/technical-task.md)
 4. [Cсылки и изображения](https://github.com/rakavets/html-coding/blob/chapter4/chapter4/technical-task.md)
-5. [Введение в таблицы](https://github.com/rakavets/html-coding/blob/chapter5/chapter5/technical-task.md)
\ No newline at end of file
+5. [Введение в таблицы](https://github.com/rakavets/html-coding/blob/chapter5/chapter5/technical-task.md)
+
```

Comment:
Delete empty string because it does not comply with the code convention

Reply:
Okey